src/extensions/report-portal-history.js

```diff
@@ -24,11 +24,20 @@ async function getLaunchHistory(extension) {
   return [];
 }
 
+function setLinkHistoryViaProperty(extension) {
```

Comment:
Instead of this function, add the new input to the default inputs object where the default value equals to **uuid** - https://github.com/test-results-reporter/reporter/blob/main/src/extensions/report-portal-history.js#L95-L98